### DISCUSSION TITLE
https://github.com/apexcharts/apexcharts.js/discussions/new

### DISCUSSION THREAD

**Discussion by wFire** [apexcharts/apexcharts.js#3275] [Q&A]
Please help with making JSON file for pie chart. I try JSON file which give that output:
`{"series":["2","5","7","8"],"labels":["One","two","five","seven"]}`
But I give a blank chart as result.

**Comment by brianlagunas** [comment]
Try one of these https://apexcharts.com/javascript-chart-demos/pie-charts/

**Comment by wFire** [comment]
This is sample code from Docs:
`var chart = new ApexCharts(el, options);
chart.updateSeries([{
  data: [32, 44, 31, 41, 22]
}])`
Please can you give the sample code, updating labels too and suitable for pie/donut chart.

**Comment by wFire** [comment]
This variant don't work:
`chart3.updateSeries([{
           name: 'edfd',
           data:[{series:"56", "34"}, {labels:"fff","gdg"}]
		  }]);`
